apps/opik-frontend/src/components/pages-shared/onboarding/CodeExecutor/useRunCodeSnippet.ts

```diff
@@ -0,0 +1,104 @@
+import { REAL_LOGS_PLACEHOLDER } from "@/components/pages-shared/onboarding/FrameworkIntegrations/integration-scripts/fake-logs";
+import { CODE_EXECUTOR_SERVICE_URL } from "@/components/pages-shared/onboarding/FrameworkIntegrations/quickstart-integrations";
+import { useState } from "react";
+
+type RunCodeRequestArgs = {
+  executionUrl: string;
+  workspaceName: string;
+  apiKey: string;
+};
+const runCodeRequest = async ({
```

Comment:
I would also break this file into 2. Since we have all the API calls and logic stored in the `src/api` folder. you could put `runCodeRequest` there